test/smoke/faucet.js

```diff
@@ -1,4 +1,20 @@
+const fetch = require('node-fetch');
+
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
 describe('Faucet', () => {
-  // getFunds
-  it('should respond positive balance');
+  networkConfig.inventory.web.hosts.forEach((nodeName) => {
+    describe(nodeName, () => {
+      it('should respond positive balance',
+        async () => {
+          const faucetBalance = await fetch(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}/`)
+            .then(res => res.text());
```

Comment:
Please avoid of mixing async/await and promises approach.

const response = await fetch()
const balance = await response.text()

Reply:
modified